gammapy/image/maps.py

```diff
@@ -341,12 +341,13 @@ def reproject(self, refheader, mode='interp', *args, **kwargs):
         skymap : `SkyMap`
             Skymap reprojected onto ``refheader``.
         """
+
         from reproject import reproject_interp
         out = reproject_interp(
             input_data=self.to_image_hdu(),
             output_projection=refheader,
             *args,
-            **kwargs,
+            **kwargs
         )
         map = SkyMap(name=self.name, data=out[0], wcs=self.wcs, unit=self.unit, meta=self.meta)
```

Comment:
you mean it shouldn't be wcs=self.wcs but wcs=reafheader , refheader being in the parameters of the method?